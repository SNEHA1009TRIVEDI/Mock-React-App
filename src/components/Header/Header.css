*{
    margin: 0;
    padding: 0;
}
header{
    background-color: #38497E;
    color: #fff;
    padding: 10px;
    padding-left: 20px;
    padding-top: 20px;
    padding-bottom: 10px;
    text-align: left;
    height: 50px;
    font-size: 20px;

}
.heading{
    color: #fff;
}

@media screen and (max-width: 768px) {
    .header {
      font-size: 15px;
      
    }
  }