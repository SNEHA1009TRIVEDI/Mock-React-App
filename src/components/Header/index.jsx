import React from 'react';
import './Header.css';
const Header = () => {
    return (
        <header className='Header'>
         <h1>My <b>Record</b> Shelf</h1>
        </header>
    );
    }

// export the component
export default Header;

