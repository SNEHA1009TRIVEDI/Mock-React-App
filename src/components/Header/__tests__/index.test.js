import { render } from "@testing-library/react";
import React from "react";
import Header from "../index";

describe("Header", () => {
  it("should renders without crashing ", () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
