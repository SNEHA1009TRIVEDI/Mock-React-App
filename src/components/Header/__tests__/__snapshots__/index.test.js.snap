// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should renders without crashing  1`] = `
<DocumentFragment>
  <header
    class="Header"
  >
    <h1
      class="heading"
    >
      My 
      <strong>
        Record
      </strong>
       Shelf
    </h1>
  </header>
</DocumentFragment>
`;
